portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,48 @@
 #greeting-container {
   margin-top: 20px;
 }
+*{
```

Comment:
Not a big deal, but generally having a space after the element (i.e. `* {`) makes it more readable :)